vtm/src/org/oscim/utils/Tessellator.java

```diff
@@ -32,20 +32,20 @@
      * Special version for ExtrusionLayer to match indices with vertex positions.
      * Tessellates polygon in tris elements.
      *
-     * @param points       the {@link org.oscim.core.GeometryBuffer#points}
-     * @param ppos         the {@link org.oscim.core.GeometryBuffer#pointNextPos} (not needed)
-     * @param numPoints    equals ppos?
-     * @param index        the {@link org.oscim.core.GeometryBuffer#index}
-     * @param ipos         the {@link org.oscim.core.GeometryBuffer#indexCurrentPos}
```

Comment:
Would it be ok to add a PR to clarify this signature?

Reply:
Sure.